js/views/pageNavVw.js

```diff
@@ -350,18 +350,19 @@ module.exports = Backbone.View.extend({
     $('#obContainer').addClass('blur');
 
     // display the modal
-    $('.js-aboutModal').removeClass('hide');
+    $('.js-aboutModal').removeClass('fadeOut').addClass('fadeIn');
   },
```

Comment:
We'll have to watch this, previously the fadeIn/fadeOut classes caused problems. 

It should also just need one class, fadeOut, removing it should do all you need.